src/main/java/org/spongepowered/api/event/entity/player/PlayerChatEvent.java

```diff
@@ -32,5 +32,18 @@
  * Called when a {@link Player} sends a chat message.
  */
 public interface PlayerChatEvent extends MessageEvent, PlayerEvent {
-
+    
+    /*
+     * Returns list of players that recieve the message.
+     *
+     * @return The list of {@link Player}s
+     */
+    List<Player> getReceivers();
+    
+    /*
+     * Sets the list of players who receive the message.
+     *
+     * @param receivers The list of {@link Player}s
+     */
+    void setReceivers(List<Player> receivers);
```

Comment:
If the list returned above is mutable, no need for a setter of them. You can grab the list and manipulate it.

Also, is ordering important to you? Otherwise, I'd use a Set.